docs/code-reviews/inclusion-in-code-review.md

```diff
@@ -46,6 +46,23 @@ Below are some points which emphasize why inclusivity in code reviews is importa
 We as CSE, may come across situations in which code reviews are not ideal and often we are observing non inclusive code review behaviors. Its important to be aware of the fact that culture and communication style of a particular geography also influences how people interact over pull requests.
 In such cases, assuming positive intent of the author and reviewer is a good start to start analyzing quality of code reviews.
 
+## Dealing with the Impostor Phenomenon
+
+Impostor phenomenon is a psychological pattern in which an individual doubts their skills, talents, or accomplishments and has a persistent internalized fear of being exposed as a "fraud" - [wikipedia](https://en.wikipedia.org/wiki/Impostor_phenomenon).
+
+Someone experiencing impostor phenomenon may find submitting a code review particularly stressful. It is important to realise that everybody can have meaningful contributions and to not let the perceived weaknesses prevent said contributions.
+
+Some tips for overcoming the impostor phenomenon for authors:
+
+* Review the guidelines highlighted above and make sure your code change adhere the them.
+* Ask for help from a colleague - pair program with an experienced colleague that you can learn from.
+
+Some tips for overcoming the impostor phenomenon for reviewers:
+
+* Anyone can have valuable insights. A fresh new pair of eyes are always welcome. Study the review until you have clearly understood it, check the corner cases and look for ways to improve it.

```

Comment:
I would split this one with * as well:
* Anyone can have valuable insights. 
* A fresh new pair of eyes are always welcome. 
* Study the review until you have clearly understood it, check the corner cases and look for ways to improve it.

Reply:
Looks better indeed.